src/Symfony/Bundle/TwigBundle/Controller/ExceptionController.php

```diff
@@ -145,4 +148,26 @@ protected function templateExists($template)
 
         return false;
     }
+
+    private function processLogs($logger)
+    {
+        $logs = array();
+        foreach ($logger->getLogs() as $log) {
+            if (class_exists('Symfony\Component\VarDumper\Dumper\HtmlDumper')) {
+                $cloner = new VarCloner();
+                $dumper = new HtmlDumper();
+                $dumper->dump($cloner->cloneVar($log['context']), $output = fopen('php://memory', 'r+b'));
+                $contextAsString = stream_get_contents($output, -1, 0);
+            } elseif (class_exists('Symfony\Component\Yaml\Yaml')) {
+                $contextAsString = Yaml::dump((array) $log['context'], 1024);
+            } else {
+                $contextAsString = json_encode($log['context'], JSON_PRETTY_PRINT);
```

Comment:
I suggest also adding the `JSON_UNESPACED_UNICODE` and `JSON_UNESCAPED_SLASHES` flags to make the output more human-readable (having `\u...` codes for non-ASCII chars makes things harder to read)